python-checks/src/test/resources/checks/methodShouldBeStatic.py

```diff
@@ -95,6 +95,10 @@ def fun23(slf): # Noncompliant
         if True:
             raise NotImplementedError
 
+    # raise issue, as the error is unlikely to be a NotImplementedError
```

Comment:
@ivandalbosco I think "unlikely" is a wrong word here:) we know for sure that it will not be NotImplementedError